Vagrantfile

```diff
@@ -21,6 +21,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provider :virtualbox do |vb|
     vb.customize ["modifyvm", :id, "--memory", "512"]
   end
+  
+  config.vm.define "el8-pam", autostart: true do |c|
```

Comment:
autostart should be false. When you do `vagrant up` without specifying a VM it will start those and we only want one to start, EL7.